components/atoms/Search/search.tsx

```diff
@@ -37,6 +42,8 @@ const Search = ({ placeholder, name, value, autoFocus, className, onSearch }: Se
           }
         }}
       />
+      { search && <GrClose className="text-light-slate-9" fontSize={16} onClick={emptySearch}/> }
```

Comment:
nit: change click handler to `handleEmpty`

```suggestion
      { search && <GrClose className="text-light-slate-9" fontSize={16} onClick={emptySearch}/> }
```